package.json

```diff
@@ -69,6 +70,7 @@
     "node-fetch": "https://github.com/scottnonnenberg/node-fetch.git#3e5f51e08c647ee5f20c43b15cf2d352d61c36b4",
     "os-locale": "^2.1.0",
     "pify": "^3.0.0",
+    "protobufjs": "^6.8.6",
```

Comment:
So we're adding a new dependency here. But I don't see any removal of our previous methods for dealing with protobufs (`protobufs` in `bower.json`). Speaking of `bower.json` I notice that it includes `preen` settings for `long` as well, though that doesn't seem to be a dependency anymore...